recipe/build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 if [ `uname` == Darwin ]; then
+    export OBJC="${CC}"
```

Comment:
Turns out `libtool` didn't know what `OBJC` was and that is [what caused issues before]( https://travis-ci.org/conda-forge/graphviz-feedstock/builds/397490964#L2235 ). The solution is to explain what compiler is needed for `OBJC`, which turns out to be the same as `CC` (a.k.a. `clang`) on macOS. With that change alone, [the build passes]( https://travis-ci.org/conda-forge/graphviz-feedstock/builds/400719599 ). Not sure why we didn't need this change before. 🤔

Reply:
Raised issue ( https://github.com/conda-forge/toolchain-feedstock/issues/40 ) about getting this into the `compiler` syntax in the future.